Doppler.BigQueryMicroservice/Repository/Classes/UserAccessByUserRepository.cs

```diff
@@ -73,6 +76,41 @@ public async Task<IReadOnlyList<UserAccessByUser>> GetAllByUserIdAsync(string ac
                 }
             }
         }
+
+        public async Task<bool> MergeEmailsAsync(string accountName, List<string> emails)
+        {
+            using (var connection = await base.CreateConnectionAsync())
+            {
+                try
+                {
+                    var dt = new DataTable("[dbo].[TypeEmail]");
+                    dt.Columns.Add("Email", typeof(string));
+
+                    foreach (var email in emails)
+                    {
+                        dt.Rows.Add(email);
+                    }
+
+                    var result = await connection.QueryAsync("EXEC [datastudio].SaveEmailsByAccounName @P1_Emails, @P2_AccounName", new
+                    {
+                        P1_Emails = dt.AsTableValuedParameter("[dbo].[TypeEmail]"),
+                        P2_AccounName = accountName
+                    });
```

Comment:
Is `SaveEmailsByAccounName` a stored procedure? You can use stored procedure execution if you want:
```suggestion
                   const string saveEmailsByAccounNameSpName = "[datastudio].SaveEmailsByAccounName";

                    connection.Execute(
                        saveEmailsByAccounNameSpName,
                        new { P1_Emails = dt.AsTableValuedParameter("[dbo].[TypeEmail]"), P2_AccounName = accountName },
                        commandType: CommandType.StoredProcedure);
```
This is an example: [https://dapper-tutorial.net/stored-procedure](https://dapper-tutorial.net/stored-procedure)

Reply:
I tried to do the work in this way but here as a table is used as parameters I used this implementation, I still got it from the official dapper documentation